guides/common/modules/ref_predefined-roles-available-in-project.adoc

```diff
@@ -0,0 +1,56 @@
+[id="Predefined_Roles_Available_in_{project-context}_{context}"]
+= Predefined Roles Available in {Project}
+
+The following table provides an overview of permissions that predefined roles in {Project} grant to a user.
+
+For a complete set of predefined roles and the permissions they grant, log in to {ProjectwebUI} as the privileged user and navigate to *Administer* > *Roles*.
```

Comment:
Should be "ProjectWebUI"; but attributes are IMHO case insensitive: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L109

cc @asteflova

Reply:
Oops, thanks, I submitted https://github.com/theforeman/foreman-documentation/pull/2397